simulator/Makefile

```diff
@@ -23,7 +23,9 @@ mod-download: ## Downloads the Go module
 .PHONY: build
 build:  
 	go build -o ./bin/simulator ./cmd/simulator/simulator.go
+	go build -o ./bin/scheduler ./cmd/scheduler/scheduler.go
 
 .PHONY: start
 start: build
 	./hack/start_simulator.sh
+	./bin/scheduler
```

Comment:
Ah, I think we can just remove start because now the simulator must be running in the docker. We cannot run it locally anymore

Reply:
I fix [11e2311](https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/350/commits/11e2311dfcedf74eb3ad1909d8ec0a76a0bccd5a).